### ISSUE TITLE
[EN][Spell][Mage] Flame orb

### ISSUE THREAD

**Issue by Zhalox123** [Atlantiss/BugTracker#9634]
**Description**: Describe the problem
When you use flame orb on a npc/raid boss and get procs to cast arcane missiles or evocation it gets interrupted by itself on orb hits while channeling.
**Current behaviour**: What happens
What happends after using flame orb on boss and you get procs of arcane missiles when you cast arcane missiles or evocation your cast gets instantly interrupted becouse of the flame orb procs or hits.
**Expected behaviour**: What should happen instead
Flame orb shouldn't break channeling of arcane missiles and evocation on hits.
**Steps to reproduce the problem**:

1. Cast Flame orb on raid boss or npc.
2. Get proc of arcane missile cast arcane missile or evocation.
3. See how arcane missiles and evocation gets interrupted.

**Revision**: Current server version 5535

**Comment by kabashx** [comment]
Same as for fire mage when u duel some1 and use flame orb your Evo is interupted with first flameorbs hit!

**Comment by Zhalox123** [comment]
Lovely bug I guess pretty new :D

**Comment by kabashx** [comment]
and u are a PvP hero, u noticed this only on PvE :P

**Comment by Zhalox123** [comment]
Yes I heard later about you and some other guy in elwyn that it interrupts also on fire evo sucks :(

**Comment by Olrios** [comment]
had it multiple times on battlegrounds and bosses
arcane missiles get canceled after 1st bolts

**Comment by stanq87** [comment]
[example1](https://www.youtube.com/watch?v=wlB9DOw6z28)
[example2](https://www.youtube.com/watch?v=2NHUfMIV2K4)
